src/components/PageHeadingArtifactKey.vue

```diff
@@ -0,0 +1,41 @@
+<template>
+  <PageHeading class="page-heading-artifact-key" :crumbs="crumbs">
+    <template #actions>
+      <ArtifactMenu :artifact="artifact" />
+    </template>
+  </PageHeading>
+</template>
+
+<script lang="ts" setup>
+  import { BreadCrumbs } from '@prefecthq/prefect-design'
+  import { computed } from 'vue'
+  import ArtifactMenu from '@/components/ArtifactMenu.vue'
+  import PageHeading from '@/components/PageHeading.vue'
+  import { useWorkspaceRoutes } from '@/compositions'
+  import { localization } from '@/localization'
+  import { Artifact } from '@/models'
+  import { pluralize } from '@/utilities'
+
+  const props = defineProps<{
+    artifact: Artifact,
+  }>()
+
+  const routes = useWorkspaceRoutes()
+
+  const crumbs = computed<BreadCrumbs>(() => {
+    const internalCrumbs: BreadCrumbs = [
+      {
+        text: pluralize(localization.info.artifact),
```

Comment:
nit: this should probably just be its own localization string.

Reply:
Agreed - added